resources/views/components/slideover/index.blade.php

```diff
@@ -0,0 +1,24 @@
+@props(['id', 'title', 'hasParent' => false, 'right' => false, 'isForm' => false])
```

Comment:
I'm not too sure what hasParent means, maybe we can clear this name up?
But i think using position, and tag or type div or form will make the usage easier to understand
```suggestion
@props(['id', 'title', 'hasParent' => false, 'position' => 'left', 'tag/type' => 'div'])
```